packages/cli/lib/lib/app.js

```diff
@@ -0,0 +1,9 @@
+import { h } from 'preact';
+import { Provider } from '../../pre-render-data';
+
+const interopDefault = m => (m && m.default ? m.default : m);
+
+export default ({ CLI_DATA: { preRenderData } = {}, ...props }) => {
+	let app = interopDefault(require('preact-cli-entrypoint'));
+	return <Provider value={preRenderData}>{h(app, props)}</Provider>;
```

Comment:
I think we still want to pass the prerender data to the root component myb?

also this still includes compat?

Reply:
Yes if we this magic `<Provider.....>` compat will be included.
I am in favor of removing the whole `<Provider value=...>` from CLI code and give it as a package to the user to use it

Reply:
sounds good